app/routes/events_.$eventId.tsx

```diff
@@ -1,11 +1,20 @@
-import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node';
+import type { MetaFunction, ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node';
 import { useParams, useLoaderData, Link, Form } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
 import { json, redirect } from '@remix-run/node';
 import { eventDataPatcher } from '~/modules/events/event';
 import { requireUserSession } from '~/modules/session/session.server';
 
-export async function loader({ params }: LoaderFunctionArgs) {
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
+  const eventFullName = data.event.name;
+  const regex = /^[^:]+/;
```

Comment:
Q: is there a reason we cleaning up only : , or should we go further and only allow a-z A-Z 0-9